lib/cru_lib/global_registry_methods.rb

```diff
@@ -8,8 +8,8 @@ module GlobalRegistryMethods
 
 
     included do
-      after_commit :push_to_global_registry
-      after_destroy :delete_from_global_registry
+      after_commit :push_to_global_registry, on: [ :create, :update ]
+      after_commit :delete_from_global_registry, on: :destroy
```

Comment:
yes

On Mon, Jun 27, 2016 at 1:14 PM, Mark Knutsen <notifications@github.com>
wrote:

> In lib/cru_lib/global_registry_methods.rb
> <https://github.com/CruGlobal/cru_lib/pull/8#discussion_r68647200>:
>
> > @@ -8,8 +8,8 @@ module GlobalRegistryMethods
> >
> >
> >      included do
> > -      after_commit :push_to_global_registry
> > -      after_destroy :delete_from_global_registry
> > +      after_commit :push_to_global_registry, on: [ :create, :update ]
> > +      after_commit :delete_from_global_registry, on: :destroy
>
> Model attributes are still available after_commit just like they are
> after_destroy, right? The callback needs global_registry_id.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/CruGlobal/cru_lib/pull/8/files/0deef896dde761c6986c048eb895d031b5b33c72#r68647200>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABcdKRk6jxTGzQ1klNo_Uv6OymA3kzcks5qQC8RgaJpZM4I_ddE>
> .
>